### PR TITLE
feedback on 02 intro

### PR DESCRIPTION
**Pull request by rasbt**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

**Any more details?**

As promised, I started revisiting the chapters that we have. Originally, I planned to go through the whole article (or at least more of the chapters) today, but it's a super busy semester, and I found the intro a bit rough so that I probably spent too much time on it. 

I made a few changes while trying to preserve the original text as much as possible. However, I felt like some sentences needed to be rephrased. Let me know what you think.

### CODE REVIEW

**Review by Benjamin-Lee — APPROVED** [review]
Looks great. Thank you again for taking the time to look over the manuscript with such a bust schedule!

File: content/02.intro.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+Machine learning is a modern approach to problem-solving and task automation. In particular, machine learning is concerned with the development and applications of algorithms that learn how to recognize patterns in data and utilize these for predictive modeling, as opposed to having domain experts developing rules for prediction tasks manually.
+Artificial neural networks are a particular class of machine learning algorithms and models that evolved into what we now describe as "deep learning" -- that is, neural networks with many layers (and algorithms that make them perform well).
+These neural networks comprise artificial neurons arranged into layers and are modeled after the human brain, even though the building blocks and learning algorithms may differ [@doi:10.1038/s41583-020-0277-3].
+Each layer receives input from previous layers (the first of which represents the input data) and then transmits a weighted version of its input to the subsequent layer. 
+Thus, the fitting or process of "training" a neural network is the tuning of the layers' weights such that minimizes a so-called cost or loss function that serves as a differentiable surrogate of the prediction error.
```
Comment (mdkessler) [inline content/02.intro.md:7]: Suggested edit-

Thus, the process of "training" a neural network is the tuning of the layers' weights to minimize a cost or loss function that serves as a differentiable surrogate of the prediction error.

**Review by mdkessler — APPROVED** [review]
I really like the changes overall! I just left a couple of suggested edits.

File: content/02.intro.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
-General resources communicating best practices to the scientific community broadly and the biological community specifically are scarce, and any resources that do exist are prone to reaching obsolescence rapidly due to deep learning's active and specialized nature.
-In addition, the lack of established standards or concise recommendations for the application of deep learning to biological questions further limits newcomers from using state-of-the-art deep learning in their research.  
+However, for researchers and students new to machine learning, and even for those experienced in using statistical techniques or classical machine learning methods (for example, generalized linear models such as linear or logistic regression and nonparametric techniques such as random forests), using deep learning correctly can be daunting due to the highly complex nature of the method and the corresponding toolkits.
+General resources communicating best practices to the scientific community broadly and the biological community are scarce, and any resources that do exist are prone to rapidly reaching obsolescence due to deep learning being an active and specialized area of research.
```
[the remark this answers is not in the document]
Comment (rasbt) [reply content/02.intro.md:14]: Sure I can change that back

File: content/02.intro.md (hunk 1 above)
Comment (rasbt) [reply content/02.intro.md:7]: I like it.
